constants/map_constants.asm

```diff
@@ -1,14 +1,15 @@
 newgroup: MACRO
 const_value = const_value + 1
```

Comment:
honestly if you're going to re-implement `const` here, just give it a more appropriate name.

Reply:
`newgroup` increments `const_value` which is used for the `GROUP` constants, and resets `__map_value__` to 1 for the subsequent `MAP` constants via `map_const`.

Are you implying it should be `group_const`? But (a) that suggests it's defining a constant, but we don't have group constants independent of the maps (e.g. it's not `group_const OLIVINE`, `group_const DUNGEONS`, etc); and (b) the name "`group_const`" wouldn't suggest anything about resetting the `map_const` counter.